bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,47 @@
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/home/home.html" %}
+
+{% add_lang_files "mozorg/home/index-quantum" %}
+
+{% block icon_links_who %}
+<ul id="icon-links-who" class="icon-links">
+  <li id="information-trust">
+    <a rel="external" href="https://blog.mozilla.org/blog/2017/08/08/mozilla-information-trust-initiative-building-movement-fight-misinformation-online/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-information-trust"></use></svg>
+      {{ _('Mozilla Information Trust Initiative') }}
+    </a>
+  </li>
+  <li id="empowering-women">
+    <a rel="external" href="https://learning.mozilla.org/blog/new-partnership-with-un-women-to-teach-key-digital-skills-to-women/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-empowering-women"></use></svg>
+      {{ _('Empowering Women Online') }}
+    </a>
+  </li>
+  <li id="coral-project">
+    <a rel="external" href="https://blog.mozilla.org/blog/2017/09/06/mozilla-washington-post-reinventing-online-comments/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-coral-project"></use></svg>
+      {{ _('The Coral Project') }}
+    </a>
+  </li>
+  <li id="nethealth">
+    <a rel="external" href="https://internethealthreport.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-nethealth"></use></svg>{{ _('Internet Health Report') }}</a>
+  </li>
+</ul>
+{% endblock %}
+
+{% block icon_links_innovate %}
+<ul id="icon-links-innovate" class="icon-links">
+  <li id="common-voice">
+    <a rel="external" href="https://voice.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-common-voice"></use></svg>{{ _('Common Voice') }}</a>
+  </li>
+  <li id="vr">
+    <a rel="external" href="https://vr.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-vr"></use></svg>{{ _('Virtual Reality Platform') }}</a>
+  </li>
+  <li id="servo">
+    <a rel="external" href="https://research.mozilla.org/servo-engines/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-servo"></use></svg>{{ _('Servo') }}</a>
+  </li>
+  <li id="rust">
+    <a rel="external" href="https://research.mozilla.org/rust/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=v-b"><svg class="icon"><use xlink:href="#icon-rust"></use></svg>{{ _('Rust') }}</a>
+  </li>
+</ul>
+{% endblock %}
```

Comment:
Is this content different than what's currently served to `en-US` visitors in `home.html`? Maybe I just can't see it...

Reply:
The only difference are the `utm` params in the links, which is something required for GA.